test/tests/static_closure_locations.py

```diff
@@ -0,0 +1,25 @@
+# should_error
```

Comment:
Can you catch the exception that gets thrown and then print it?  #should_error is undesirable because then we can't detect a mistake in the test itself (ex a typo).